### PR TITLE
Avoiding a double process when launching a workflow using the protocol form

### PR DESCRIPTION
**Pull request by fonsecareyna82**
* This occurs when a form is used to launch a subworkflow from the protocol.

### CODE REVIEW

**Comment by azazellochg** [comment]
Please have a look at sonarcloud issue

File: CHANGES.txt [hunk 1]
```diff
@@ -3,12 +3,14 @@ V3.6.1
 user:
   - Streaming protocols based on new streaming (generation steps) are resumable
   - pyworkflowtest fixed
+  - Avoiding a double process when launching a workflow using the protocol form
```
Comment (pconesa) [inline CHANGES.txt:6]: May be you can comment on the new param "executable" in runJob method?

Comment (delarosatrevin) [reply CHANGES.txt:6]: I don't get the point/need of this new parameters to runJob (and the rest of related functions)

Comment (pconesa) [reply CHANGES.txt:6]: This has been triggerred by the fact that warp has a "bash" script that is called when activating the environment.

We always call sh, and the way to call bash is to either:

make the command bash + arguments or offer this option.

Warp activation environment does not work on sh shells.

Comment (delarosatrevin) [reply CHANGES.txt:6]: Ok, in my opinion, it would be easier to just call bash as the first argument without introducing a new parameter.

**Review by danielmarchan3 — APPROVED** [review]
As it is, with the same example that we used to demonstrate the issue, it doesn't happen anymore. However, I agree that if we can resolve it without adding a parameter, that would be preferable. If not, this fix addresses the bug. Thank you Yunior :)

File: CHANGES.txt (hunk 1 above)
Comment (pconesa) [reply CHANGES.txt:6]: I know but the code is clearer like WarpTools as a program instead of being the program bash.

Still, developers can do both now.

**Comment by pconesa** [comment]
@danielmarchan3 the extra parameter is unrelated to the issue you tested. It is just that was convenient to use the same PR to fix 2 issues.

**Pull request merged**